components/ws-manager/pkg/manager/manager.go

```diff
@@ -752,12 +752,15 @@ func (m *Manager) stopWorkspace(ctx context.Context, workspaceID string, gracePe
 
 	span, ctx := tracing.FromContext(ctx, "stopWorkspace")
 	defer tracing.FinishSpan(span, &err)
+	span.LogKV("workspaceID", workspaceID)
 
 	pod, err := m.findWorkspacePod(ctx, workspaceID)
 	if isKubernetesObjNotFoundError(err) {
+		tracing.LogError(span, err)
```

Comment:
[this should be logged in FinishSpan](https://github.com/gitpod-io/gitpod/blob/16cdbb02225232f156097fc7d71a3730a8a6fdf3/components/common-go/tracing/tracing.go#L90)

Reply:
good catch! fixed!